src/Core/Module/ModuleManager.php

```diff
@@ -229,6 +231,15 @@ public function upgrade(string $name, $source = null): bool
 
         $this->hookManager->exec('actionBeforeUpgradeModule', ['moduleName' => $name, 'source' => $source]);
 
+        $this->log(
```

Comment:
Why is this the only log you added in the module manager while the others are all in the legacy class? Is this because you need to log the beginning and the end of the upgrade and it was the only way to do this? 🤔

Reply:
@jolelievre because the module upgrade process is there and not in the legacy class. Install/uninstall are in the legacy class.

Reply:
Hmm it's not exactly true, you still have those methods in the legacy class:
- initUpgradeModule
- runUpgradeModule
- upgradeModuleVersion
- onUpgrade

You even added logs in the `runUpgradeModule`, that's why I wondered if it could all be in the legacy class only?

Reply:
runUpgradeModule in legacy module is not called by the controller, that' why I've added it in ModuleManager